### ISSUE TITLE
Always installing compiler version

### ISSUE THREAD

**Issue by revodavid** [RevolutionAnalytics/checkpoint#156]
Whenever I run the attached script, I get the following message:

```
* installing *binary* package ‘compiler’ ...
* DONE (compiler)
Scanning for loaded pkgs
Scanning for packages used in this project
Installing packages used in this project 
Warning: dependencies ‘utils’, ‘stats’, ‘methods’, ‘grid’ are not available

Attaching package: ‘dplyr’
... [DELETED]
```

Not sure why the message about `compiler` appears. Oddly, it doesn't appear in `search()` after the script completes.
[weather.R | uploaded via ZenHub](https://files.zenhub.io/54bff31ef748cd180f07a431)

**Comment by revodavid** [comment]
Using the current dev version, BTW.

**Comment by andrie** [comment]
Yes, this is desired behaviour and was necessary to fix #143
